CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [v0.6.6] - 2015-05-12
```

Comment:
You can add a link for v0.6.6 on line 230. Just look at the existing convention for the previous versions and adapt it accordingly.